### ISSUE TITLE
web cannot access local file.

### ISSUE THREAD

**Issue by cyilu** [PAIR-code/deeplearnjs#908]
As the security of web, it can't access local file. Therefore, it's difficult to train in the js.

Any help will be appreciate！

**Comment by tafsiri** [comment]
You may need to set up a local server to serve your files. Take a look at https://developer.mozilla.org/en-US/docs/Learn/Common_questions/set_up_a_local_testing_server

**Comment by cyilu** [comment]
@tafsiri Yes, thank you! But it means that I can use local file in the browser?

**Comment by tafsiri** [comment]
Yes

**Comment by dsmilkov** [comment]
This issue was moved to tensorflow/tfjs#63
